src/Data/Food/Ingredient.elm

```diff
@@ -37,6 +38,11 @@ type Id
     = Id String
 
 
+byPlaneFromOrigin : Origin.Origin -> Bool
+byPlaneFromOrigin origin =
+    origin == Origin.OutOfEuropeAndMaghrebByPlane
```

Comment:
Nit: I'd rather have the full ingredient passed as the argument, and maybe the function renamed: 

```elm
if Ingredient.byPlaneByDefault mango then
``` 

reads a little better than 

```elm
if Ingredient.byPlaneFromOrigin mango.defaultOrigin then
```